libzeth/core/ff_utils.hpp

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2015-2020 Clearmatics Technologies Ltd
```

Comment:
Seems like you renamed `sciprlab_libs_utils` to `ff_utils`. These files do not manipulate finite field elements though, but also elliptic curve points, so the name doesn't really reflect the content

Reply:
`ff` was supposed to reflect `libff`, but I agree the name is not great.  I was trying to avoid using the name of dependent libraries or groups in these file names, so looking for something generic (hence `ff_utils` was also not satisfying).   I could separate it into `field_element_utils` and `group_element_utils`?  Or can you think of a better name?

Reply:
The only name that comes to my mind is `ff_ec_utils.*`... I think the separation `field_element_utils` and `group_element_utils` may be better